mutiny-core/src/nostr/mod.rs

```diff
@@ -473,15 +477,28 @@ impl<S: MutinyStorage> NostrManager<S> {
                 })
                 .collect();
             let builder = EventBuilder::new(Kind::ContactList, json!(content).to_string(), tags);
-            self.client.send_event_builder(builder).await?;
+            let event = self
+                .nostr_keys
+                .read()
+                .await
+                .signer
+                .sign_event_builder(builder)
+                .await?;
+
+            // don't broadcast if we're testing
+            #[cfg(not(test))]
+            self.client.send_event(event.clone()).await?;
```

Comment:
I don't like these, there's no reason it should have errors in CI doing this.

Especially if you're already doing mock stuff anyways. Either let them broadcast or mock them out, but don't comment out the code for the sake of tests.

Reply:
still gotta remove them